docs/v5/configuration/htcondor-routes.md

```diff
@@ -14,7 +14,7 @@ Setting periodic hold or release
 
         SYSTEM_PERIODIC_REMOVE = $(SYSTEM_PERIODIC_REMOVE) || <YOUR REMOVE EXPRESSION>
 
-To release or put routed jobs on hold if they meet certain criteria, use the `Periodic_*` family of attributes.
+To release or put routed jobs on hold if they meet certain criteria, use the `Periodic*` family of attributes.
```

Comment:
All these `Periodic` attr related changes seem irrelevant to the ticket. In the future, we should at least split this out into a different commit.